### PR TITLE
React

### LINKED ISSUE #13: Add separator for alt names
Alt names should be comma-separated.

### PR DESCRIPTION
**Pull request by hilolo**
## For What?
I've added a separator for alt names

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ export default class App extends Component {
     return (
       <React.Fragment>
         <nav className="navbar sticky-top navbar-light bg-light px-5">
-          <a className="navbar-brand">Anusic React</a>
+          <a href="/#" className="navbar-brand">Anusic React</a>
```
Comment (eoussama) [inline src/App.js:24]: Will this cause any navigation errors?

File: src/App.js [hunk 2]
```diff
@@ -58,17 +58,16 @@ export default class App extends Component {
                 <a
                   className="list-group-item list-group-item-action"
                   href={`https://myanimelist.net/anime/${e.id}`}
-                  target="_blank"
-                  key={i}
+                  target="_blank" rel="noreferrer"
```
Comment (eoussama) [inline src/App.js:61]: Also, is this supposed to fix the console warnings?

**Review by eoussama — APPROVED** [review]

File: src/App.js (hunk 2 above)
Comment (hilolo) [reply src/App.js:61]: Yes , all warning issues are resolved now

File: src/App.js (hunk 1 above)
Comment (hilolo) [reply src/App.js:24]: Nop , all working fine

**Pull request merged**